src/Angular.js

```diff
@@ -1444,6 +1444,26 @@ function getNgAttribute(element, ngAttr) {
   return null;
 }
 
+function allowAutoBootstrap(document) {
+  if (!document.currentScript) {
+    return true;
+  }
+  var src = document.currentScript.getAttribute('src');
+  var link = document.createElement('a');
+  link.href = src;
+  var scriptProtocol = link.protocol;
+  var docLoadProtocol = document.location.protocol;
+  if ((scriptProtocol === 'resource:' ||
+       scriptProtocol === 'chrome-extension:') &&
+      docLoadProtocol !== scriptProtocol) {
```

Comment:
Because that's the intended use case - an extension that has a page (within the extension) that loads angular, and contains `ng-app`. We could disable that, but if we support auto bootstrap, I think we should support it within extensions, shouldn't we?